### ISSUE TITLE
Default values and missing variables

### ISSUE THREAD

**Issue by benwilson512** [absinthe-graphql/absinthe#102]
Suppose:
```
field :thing, :thing do
  arg :id, :id, default_value: "1"
end

query Foo($id: ID) {
  thing(id: $id)
}
```
If you don't pass in an `id` variable the default is not selected.

**Comment by bruce** [comment]
@benwilson512 Take a peek at this; believe this is handled in the v1.2 branch (if so, close it).

**Comment by bruce** [comment]
@benwilson512 Confirm, this is being handled in #160.

**Comment by benwilson512** [comment]
Fixed in v1.2 beta 0
